janusgraph-hbase/src/main/java/org/janusgraph/diskstorage/hbase/HBaseStoreManager.java

```diff
@@ -540,8 +540,14 @@ public boolean exists() throws BackendException {
     public List<KeyRange> getLocalKeyPartition() throws BackendException {
         List<KeyRange> result = new LinkedList<>();
         try {
-            ensureTableExists(
-                tableName, getCfNameForStoreName(GraphDatabaseConfiguration.SYSTEM_PROPERTIES_STORE_NAME), 0);
+            if (skipSchemaCheck) {
+                logger.debug("Skipping schema check");
+                exists();
```

Comment:
Question: when we skip schema check, what is the point of calling `exists()` method?

Reply:
I guess the purpose of this parameter is to skip all DDL on hbase when `skip-schema-check=true` assuming user do not have create/admin privilege. In this can we assume that the tables are already present and all column families are created beforehand. So this check is just to verify the existence of the table and fail early if it is not present

Reply:
Got it. In that case should we throw an exception if `exists()` returns false? Otherwise, calling this method seemingly does nothing.

Reply:
@jigs1993 will you be able to change the logic of this case to throw an exception as suggested by @li-boxuan ? Unless you have other thoughts about it

Reply:
@li-boxuan @porunov thanks for pointing this out! I will update it to throw the exception if table doesn't exist